Upgrade cassandra version to 3.11.2 so it does not break on new jdk v…

robinvandenberg opened the pull request:
Can somebody take a look at it? 

This project is kind off new for me. (haven't looked into it in detail)
The tests even fails for me when I checkout master. 
Also my IntelliJ isn't able to even compile the project. (mvn runs fine)
Thus, i'm not sure if these are all the required changes..

doanduyhai commented:
`Also my IntelliJ isn't able to even compile the project. (mvn runs fine)` --> RTFM

robinvandenberg commented:
> RTFM  
I did. After I needed to follow an instruction manual to setup my IDE for a regular java project and it still doesn't work, I gave up.
The 'common', 'embedded' and 'model' module run fine. it's the 'core'-module that my intelliJ doesn't want to compile.
(yes i tried mvn clean compile, yes i enabled annotation processing, yes i added generated-sources/annotations to production sources, yes i deleted the 'exclude' on the /target folder, although after some time IntelliJ reverts that)
